src/nycdb/datasets/rentstab_v2.yml

```diff
@@ -1,8 +1,8 @@
 ---
 files:
   -
-    url: https://s3.amazonaws.com/justfix-data/rentstab_counts_for_pluto_19v1_bbls.csv
-    dest: rentstab_counts_for_pluto_19v1_bbls.csv
+    url: https://s3.amazonaws.com/justfix-data/rentstab_v2.csv
+    dest: rentstab_v2.csv
```

Comment:
Hmm, the name of this file seems to imply that it won't change, e.g. that when we create a new CSV next year, we will be overwriting this `rentstab_v2.csv`, and it will have two new columns, `uc2020` and `pdfsoa2020`?

If that's true, I'd argue that it might be better to give it a different name for each time the columns change... If someone is using a slightly old version of nycdb for whatever reason, it would be good if the CSV it retrieved had exactly the columns it expected, since I think that nycdb raises exceptions if it runs into any columns it doesn't know about.

What do you think about calling this `rentstab_v2_2019.csv` or something?  And then next year's can be `rentstab_v2_2020.csv`?

Reply:
hi @toolness and @sraby . Thank for all your work! 

@toolness Either nane here  is fine with me. It's true NYCB should raise an error if there's a mismatch of columns (at some point it would be nice to improve those error messages when this happens).

Relatedly, what do you think about renaming this dataset `rentstab_doffer` or `rentstab_justfix_doffer` (it's source is https://github.com/justFixNYC/nyc-doffer correct?). We could also rename the older rent-stabilization dataset, `rentstab_taxbills`, to better indicate it's source.

Reply:
hey @toolness @aepyornis, let me know what you think of my comments above! I'd love to get the data merged in ASAP so let me know if you'd prefer to have a phone call or something to help talk things through

Reply:
oh no issues with merging from me here, go ahead. It also makes sense to keep the names unchanged if changing them would break a bunch of other programs.

Reply:
Ok, no worries @aepyornis. And I finally understand what you mean @toolness— you're just suggesting a change to the actual csv name, not the table. In that case, I'd suggest naming the source CSV as `rentstab_counts_from_doffer_2019.csv`. That way, we can specifically spell out the most recent year we have data for, and also indicate the source a bit better. I know this isn't as clear as indicating source in the table name itself, but hopefully is somewhat more clear?